### DISCUSSION TITLE
Power ups

### DISCUSSION THREAD

**Discussion by DamienAmbrose** [Default-Cube-Studios/Movement-Template#21] [Ideas]
Hi everyone! It's Damien again. We decided to build-in _power ups_ to make the damage system more forgiving. Currently, we're looking into `ScriptableObject` classes, but we still need ideas on how power ups work and what kind of effects they have on the player.
As always, all ideas are welcome!

**Comment by adityamallkush** [comment]
Powerups may **increase player health**, by a small amount of course, and they may give players **shields/extra health**
